src/Hosting/test/FunctionalTests/ShutdownTests.cs

```diff
@@ -24,14 +24,14 @@ public class ShutdownTests : LoggedTest
                                                             "Stopped end";
 
         [ConditionalFact]
-        [OSSkipCondition(OperatingSystems.Windows | OperatingSystems.MacOSX)]
```

Comment:
Why did this change? This should be a Linux only test.

Reply:
Because all of the Unix builds are failing in `dotnet-install`, I just wanted to see if it'd pass CI. I'm trying https://github.com/aspnet/Extensions/pull/2737/commits/5c45866965a2e02cf6ba617fe77dd01319e28626 to get around the dotnet-install failure. Going to revert both before I merge this.